content/blog/eng/from-0-to-infinite-the-final-keyword.md

```diff
@@ -0,0 +1,333 @@
+---
+authors: ["bruce"]
+date: "2018-12-04"
+draft: true
+share: false
+categories: [English, php, oop]
+title: "From 0 to infinite the Final keyword"
```

Comment:
Maybe we can reword the tile like this?
```suggestion
title: "From 0 to infinite: the Final keyword"
```

Reply:
Also, for consistency: `0` or `zero`, but the same everywhere. Maybe the latter is better?